tests/manual/watchdog.html

```diff
@@ -0,0 +1,15 @@
+<button id="error-1">Simulate a 1st editor crash</button>
+<button id="error-2">Simulate a 2nd editor crash</button>
```

Comment:
It would be cool to add a plugin which crashes editor when a specific letter is typed instead of external buttons. I think it would simulate crash situation better, but I do not think it will change how this test works.